test/simulation.jl

```diff
@@ -0,0 +1,197 @@
+import Gurobi
+import JSON
+import JuMP
+using WaterModels
+using PyCall
+
```

Comment:
Any modules that are needed should be imported (not used) in `runtests.jl` not in the sub-test files.  Shortcuts for WaterModels and InfrastructureModels are already defined there.